### DISCUSSION TITLE
batch_size in Sionna

### DISCUSSION THREAD

**Discussion by weirdooo-18** [NVlabs/sionna#346] [Q&A] [answered]
What does the batch_size refer to exactly? How does the number of batches given become the number of receivers (in Site-specific link level simulations)?
![55555](https://github.com/NVlabs/sionna/assets/161753847/11f37c6b-5a69-45fc-bcd8-10869d91d52d)

**Comment by SebastianCa** [comment, marked as answer]
The function `sample_positions` allows to sample `batch_size` random user positions from the coverage map. In this case, `batch_size` denotes the number of samples that are returned by the function (i.e., a tensor of shape `[batch_size, 3]` containing random `(x,y,z)` positions). 

You can find a detailed example in the [Introduction to Sionna RT](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#Site-specifc-Link-Level-Simulations) notebook.
